### PR TITLE
slackcat 1.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by DomT4** [comment]
Any resource updates needed? Otherwise 👍.

**Comment by ilovezfs** [comment]
@DomT4 PR refreshed.

**Comment by DomT4** [comment]
👍

File: Formula/slackcat.rb [hunk 1]
```diff
@@ -17,47 +17,42 @@ class Slackcat < Formula
...
     ENV["GOPATH"] = buildpath
-    ENV["GOHOME"] = buildpath
-
-    mkdir_p buildpath/"src/github.com/vektorlab/"
-    ln_sf buildpath, buildpath/"src/github.com/vektorlab/slackcat"
     Language::Go.stage_deps resources, buildpath/"src"
-
-    system "go", "build", "-ldflags", "-s -X main.version=#{version}", "-o", bin/"slackcat"
+    system "go", "build", "-o", bin/"slackcat",
+           "-ldflags", "-X main.version=#{version}"
```
Comment (ilovezfs) [inline Formula/slackcat.rb]: @DomT4 BTW this is copied from the style of equivalent wrappings in other go formulae but it looks bizarre to me. If we're going to force semantic alignment, shouldn't this line begin under the `"-o",` not under `"go"`?

Comment (DomT4) [inline Formula/slackcat.rb]: Generally, no. We align things with the first "argument" after `system` where practical. If it makes you happier:
```ruby
 system "go", "build",
        "-o", bin/"slackcat",
        "-ldflags", "-X main.version=#{version}"
```
Could also be valid.

Comment (ilovezfs) [inline Formula/slackcat.rb]: I'm thinking of
```
    system "./configure", "--disable-dependency-tracking",
                          "--prefix=#{prefix}"```

**Pull request closed without merging**